### ISSUE TITLE
Feature Request: Integration with Sprout Forms

### ISSUE THREAD

**Issue by mikestecker** [elivz/VzAddress-Craft#34]
It would be cool to get some integration with Sprout Forms and dynamically rendering the fields in a template: https://sprout.barrelstrengthdesign.com/craft-plugins/forms/docs/customization/custom-front-end-fields

**Comment by elivz** [comment]
Good idea. I haven't used Sprout Forms much myself, but I will look into adding this for v2 (Craft3-compatible).

**Comment by mikestecker** [comment]
Cool thanks, yeah it currently shows up as an available field type when you build a form with Sprout Forms but just doesn't show on the frontend.
